### PR TITLE
Update _1List.java

### PR DESCRIPTION
**Pull request by Taher1985**

### CODE REVIEW

**Comment by Taher1985** [comment]
Please approve

File: com/list/_1List.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 public class _1List {
 
 	public void listOperation() {
-		List<String> names = Arrays.asList("A", "B", "C", "D", "E", "F");
+		List<String> names = Arrays.asList("A", "B", "C", "D", "E", "F", "G");
```
Comment (Taher2011) [inline com/list/_1List.java:9]: G looks good

Comment (Taher1985) [reply com/list/_1List.java:9]: Thanks

**Review by Taher2011 — APPROVED** [review]

**Comment by Taher2011** [comment]
approved

**Pull request merged**